imagery/imageryintro.html

```diff
@@ -86,6 +86,19 @@ <h3>Imagery import</h3>
 reference points the scanned map can be rectified to obtain geocoded
 data.
 
+<h3>Semantic label information</h3>
+
+Semantic labels are a description like band names for satellite imagery
+which can be stored as metadata. <a href="r.semantic.label.html">r.semantic.label</a>
+allows assigning a semantic label information to a single raster map
+or to a list of specified raster maps.
+To print available semantic labels used for multispectral data, use
+<a href="i.band.library.html">i.band.library</a>.
+Specific band reference can be assigned to a raster map as a semantic
+label by the modules <em><a href="r.semantic.label.html">r.semantic.label</a></em>
+or <em><a href="r.support.html">r.support</a></em>. Semantic labels
```

Comment:
This sounds repetitive with the second sentence. AFAIU, r.semantic.label allows only to add the satellite imagery band references defined in i.band.library (see the Notes section in https://grass.osgeo.org/grass80/manuals/r.semantic.label.html), while r.support allows to add any sort of semantic label the user wishes. 

Maybe the info and figure I added in [New Features page](https://trac.osgeo.org/grass/wiki/Grass8/NewFeatures80) could be useful here.

Reply:
There's a figure already in raster3dintro (https://grass.osgeo.org/grass78/manuals/raster3dintro.html). Is it complicated to store figs for intros? In my opinion, figures are helpful to explain concepts.

Reply:
Most things are doable :) See 914ff2c671ba7b0f007fda934c2a5ee79b0ee2b4

Remaining issue: I don't understand how the copying of `lib/raster3d/raster3d_layout.png` works to replicate it for `lib/imagery/band_references_scheme.png`? Where is the Makefile magic hidden?

Reply:
OK, got it, that image is installed by a raster3d module and just happens to be in the right place.